### PR TITLE
feat(examples): Add labels for scale-to-zero

### PR DESCRIPTION
**Pull request by razvand**
Add labels for scale-to-zero / stateful configuration in each Kraftfile in examples. Some had already been configured to use scale-to-zero / stateful. Do not update those.

### CODE REVIEW

File: postgres/Kraftfile [hunk 1]
```diff
@@ -4,6 +4,11 @@ name: postgres
 
 runtime: base-compat:latest
 
+labels:
+  cloud.unikraft.v1.instances/scale_to_zero.policy: "on"
+  cloud.unikraft.v1.instances/scale_to_zero.stateful: "false"
+  cloud.unikraft.v1.instances/scale_to_zero.cooldown_time_ms: 1000
```
Comment (marcrittinghaus) [inline postgres/Kraftfile:10]:
```suggestion
  cloud.unikraft.v1.instances/scale_to_zero.policy: "idle"
  cloud.unikraft.v1.instances/scale_to_zero.stateful: "true"
  cloud.unikraft.v1.instances/scale_to_zero.cooldown_time_ms: 3000
```

File: redis/Kraftfile [hunk 2]
```diff
@@ -2,6 +2,11 @@ spec: v0.6
 
 runtime: redis:latest
 
+labels:
+  cloud.unikraft.v1.instances/scale_to_zero.policy: "on"
+  cloud.unikraft.v1.instances/scale_to_zero.stateful: "false"
+  cloud.unikraft.v1.instances/scale_to_zero.cooldown_time_ms: 1000
+
```
Comment (marcrittinghaus) [inline redis/Kraftfile:9]:
```suggestion
  cloud.unikraft.v1.instances/scale_to_zero.policy: "idle"
  cloud.unikraft.v1.instances/scale_to_zero.stateful: "on"
  cloud.unikraft.v1.instances/scale_to_zero.cooldown_time_ms: 1000
```

File: wordpress-all-in-one/Kraftfile [hunk 3]
```diff
@@ -4,6 +4,11 @@ name: wordpress
 
 runtime: base-compat:latest
 
+labels:
+  cloud.unikraft.v1.instances/scale_to_zero.policy: "on"
+  cloud.unikraft.v1.instances/scale_to_zero.stateful: "false"
+  cloud.unikraft.v1.instances/scale_to_zero.cooldown_time_ms: 1000
```
Comment (marcrittinghaus) [inline wordpress-all-in-one/Kraftfile:10]:
```suggestion
  cloud.unikraft.v1.instances/scale_to_zero.policy: "on"
  cloud.unikraft.v1.instances/scale_to_zero.stateful: "true"
  cloud.unikraft.v1.instances/scale_to_zero.cooldown_time_ms: 1000
```

**Comment by marcrittinghaus** [comment]
For the `node` examples, I don't know regarding `stateful`. But for `nextjs` it is definitely needed, due to JIT. It is thus likely to be also needed for the other `node` examples.

**Comment by razvand** [comment]
> For the `node` examples, I don't know regarding `stateful`. But for `nextjs` it is definitely needed, due to JIT. It is thus likely to be also needed for the other `node` examples.

I added `stateful` as `true` for `nextjs` and `idle` for `policy`. Is that OK?

**Review by marcrittinghaus — APPROVED** [review]
Reviewed-by: Marc Rittinghaus <marc.rittinghaus@unikraft.io>
Approved-by: Marc Rittinghaus <marc.rittinghaus@unikraft.io>

**Pull request merged**